### ISSUE TITLE
Add label to geodata importer

### ISSUE THREAD

**Issue by dirkseidensticker** [eScienceCenter/Spacialist#332]
At the moment importing geodata adds only the geometries to the DB. To review which geometry is associated with any specific context involves an external tool like a GIS in which you can activate labels. It would facilitate the integration of large sets of geodata imported though the GIS module if you could assign a label field that is then displayed once you over hover a unlinked geometry (unlike 'geodate ###').

**Comment by v1r0x** [comment]
Instead of simply displaying a label, the imported geodata should be linked to existing contexts. But this is not that easy to implement, because labels/names don't have to be unique in Spacialist.
Adding a parent label as well to the data could decrease the collisions of names, but you could still have `Context 1 > Room 1 > Object 1` and `Context 2 > Room 1 > Object 1`. So, neither `Object 1` nor `Room 1 > Object 1` is unique.

If you (or someone else) has a good idea how to avoid collisions I could implement it.

**Comment by dirkseidensticker** [comment]
Hm, I'll think about it. You are right, checking if an associated context exists and of not creating a new on would be the best solution.

The other day we discussed our workflow here: with respect to that it would be great if the importer would create concepts from the geodata. Comming back from the field we would import the GPS points we created.

But I guess then we would need details like datatypes etc. within the imported geodata to work somewhat like the data importer.

Let me think about it a bit more and thanks for the swift reply.
